.vscode/launch.json

```diff
@@ -14,13 +14,14 @@
             "args": [     
                 "-r",
                 "ts-node/register",                            
-                "${workspaceRoot}/CheckArtifactConsistencyTask/tests/**/*.ts"            
+                "${workspaceRoot}/CheckArtifactConsistencyTask/tests/*.ts"            
             ],            
             "cwd": "${workspaceRoot}/CheckArtifactConsistencyTask",   
             "internalConsoleOptions": "openOnSessionStart",
             "sourceMaps": true,
             "diagnosticLogging": true,
-            "preLaunchTask": "build - CheckArtifactConsistencyTask"
+            "preLaunchTask": "build - CheckArtifactConsistencyTask",
+            "autoAttachChildProcesses": true
```

Comment:
This is the ticket for the debugging. This attaches to the child_process spawned shell (the MockTestRunner does this)